lib/ts/recipe/session/sessionContext.ts

```diff
@@ -1,14 +1,9 @@
 import React from "react";
 import { SessionContextType } from "./types";
 
-const SessionContext = React.createContext<SessionContextType & { isDefault: boolean }>({
+const SessionContext = React.createContext<SessionContextType & { isDefault?: boolean }>({
```

Comment:
why is isDefault optional? I thought that it will be used by SessionAuth to know that if it's already enclosed by a parent SessionAuth or not, and also that the user exposed context type will not have this. So for internal use, shouldn't we keep isDefault compulsory?

Reply:
- it makes typing it properly in `SessionAuth` easier (it could be done I think, but more work)
- we are checking strict equality anyway.